### PR TITLE
Prepare release 0.10.1.

### PR DESCRIPTION
**Pull request by benjyw**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Release Notes
 
+## 0.10.1
+
+This release fixes an issue where some Mac machines were unable to install Pants due to a missing
+`--find-links` argument.
+
```
[the remark this answers is not in the document]
Comment (benjyw) [reply CHANGES.md:7]: I believe it is the only one worth calling out. Do you see any others?

[the remark this answers is not in the document]
Comment (thejcannon) [reply CHANGES.md:7]: https://github.com/pantsbuild/scie-pants/commit/4df586c25f8698a1734dedf0c8351af249c1f2d3

And

https://github.com/pantsbuild/scie-pants/commit/891972ba3cb75f2bb7fe383927f3f25822e4c826 

Probably

[the remark this answers is not in the document]
Comment (benjyw) [reply CHANGES.md:7]: I literally copied your notes from your release attempt!! :)

[the remark this answers is not in the document]
Comment (benjyw) [reply CHANGES.md:7]: I don't think #260 is of interest to end users, but #261 is.

**Review by huonw — APPROVED** [review]

**Pull request merged**